src/app/events.tsx

```diff
@@ -16,47 +16,92 @@ type EventCardProps = {
 };
 
 const Events = () => {
-    const EventCard: React.FC<EventCardProps> = ({ communityname, title, date, location, venue, link, logo }) => (
-        <a
-        href={link}
-        target="_blank"
-        rel="noopener noreferrer"
-        className="cursor-pointer hover:shadow-md transition-shadow border rounded-lg group"
-    >
-        <div className="bg-white rounded-lg border border-gray-200 p-4 shadow-sm relative">
-            {logo && (
-                <div className="absolute top-3 right-3">
-                    <Image 
-                        src={logo} 
-                        alt={`${communityname} logo`} 
-                        className="w-6 h-6 rounded-full filter grayscale group-hover:filter-none transition-all duration-300 object-cover"
+    const EventCard: React.FC<EventCardProps> = ({ communityname, title, date, location, venue, link, logo }) => {
```

Comment:
Looks amazing as a special effect, would be really great if we could hide the gray border onHover and then have this effect so that both of those borders dont stack up. 

This is minor and can be solved in another PR too, but just putting this in